dguta/dguta_test.go

```diff
@@ -459,10 +459,10 @@ func TestDGUTA(t *testing.T) {
 							err = db.Open()
 							So(err, ShouldBeNil)
 
-							ds, errd := db.DirInfo("/", nil)
+							ds, errd := db.DirInfo("/", &Filter{})
 							So(errd, ShouldBeNil)
-							So(ds.Count, ShouldEqual, 407)
-							So(ds.Size, ShouldEqual, 93684)
+							So(ds.Count, ShouldEqual, 33)
```

Comment:
Was the 407 a bad test expectation change that was multi-counting the same files? Or why has this dropped to 33?

Reply:
The filter used to be `nil` meaning nothing was being filtered and we assumed this isn't a real situation. Now it's a blank filter and the age is being considered so there's less values.